### PR TITLE
Added routes

### PR DESCRIPTION
**Pull request by svhnstorage**
Added createInstance route, tweaked instances to return all instances from db

### Frontend:
- None
### Backend:
- app.py

### CODE REVIEW

**Review by joshcwinton — APPROVED** [review]

File: backend/app.py [hunk 1]
```diff
@@ -175,7 +177,13 @@ def history():
 @app.route("/instances", methods=["GET"])
 def instance_list():
     if request.method == "GET":
-        return jsonify({"instances": getInstances()})
+        return jsonify({"instances": getAllStoredInstances('instances')})
+    return None
+
+
+@app.route('/createinstance/<worldName>/<creator>/<size>/')
```
Comment (joshcwinton) [inline backend/app.py]: I think for this one we can add the data as a request payload like we do with the chat ie request.get_json()

Comment (joshcwinton) [reply backend/app.py]: this is perfect for getting a single instance though

**Review by joshcwinton — CHANGES_REQUESTED** [review]

File: backend/app.py (hunk 1 above)
Comment (joshcwinton) [reply backend/app.py]: actually nvm ill merge it how it is and then see if it works

Comment (JasperCheung) [reply backend/app.py]: it should be json i think

Comment (JasperCheung) [reply backend/app.py]: just have `/createinstance'` and read from request body

**Review by joshcwinton — APPROVED** [review]
money

**Review by JasperCheung — APPROVED** [review]
this is big

**Pull request merged**